bin/src/Flags.purs

```diff
@@ -191,9 +191,7 @@ pursArgs =
       )
 
 execArgs :: Parser (Maybe (Array String))
-execArgs =
-  OT.optional
-    $ Array.fromFoldable
+execArgs = (\s -> if List.null s then Nothing else (Just $ Array.fromFoldable s))
```

Comment:
Just a thought, but why not just use `Array String` as the result here rather than `Maybe (Array String)`? `[]` is essentially the same as `Nothing`.

Reply:
Ah, just saw the issue. `Maybe` indicates whether this should override the configuration file's value.

Reply:
yeah I was also thinking that whether it would be better if we append both of them, that i have my default args in my spago.yaml, still I can pass extra arguments explicitly while doing `spago run`... but it's possible that it might cause confusion if users are not aware of this behaviour...

Reply:
Yes, the general convention in Spago is that any flag provided on the command line overrides whatever is in the config, so we shouldn't use both.

Reply:
got it.